dicom-archive/get_dicom_info.pl

```diff
@@ -466,12 +466,15 @@ =head3 InfoOption(@addr)
 
 Greps the group and element information from the GetOpt table options specified.
 
-INPUTS: array with group and element information from the GetOpt table.
+INPUTS:
+  - $option: name of the option
+  - $rest  : array with group and element information from the GetOpt table
```

Comment:
reference sur le reste des arguments du command line

Dernier arg: group and element du DICOM